### ISSUE TITLE
conflict between tags with multiple attribute and mn-dialog

### ISSUE THREAD

**Issue by rodrigodobbin** [reserva-facil/minimalist#68]
Case:

- Use mn-input or mn-select with multiple attr inside a mn-dialog.
- Digit anything you want to make it a tag
- When you press enter the tag is saved in your component
- BUT it closes the dialog as well

The dialog component should respect the listeners of the components inside him.

**Comment by darlanmendonca** [comment]
yeah, we need listen events on inputs, and dont propagate theses for mn-dialog, while user interact with inputs

**Comment by darlanmendonca** [comment]
this bug dont happening only with attribute multiple, but with every similar event between component inside a mn-dialog

**Comment by rodrigodobbin** [comment]
Okay, then it is a bigger bug than i thought.

**Comment by darlanmendonca** [comment]
yes
